akka-camel/src/test/scala/akka/camel/internal/component/ActorComponentConfigurationTest.scala

```diff
@@ -17,11 +17,11 @@ class ActorComponentConfigurationTest extends WordSpec with Matchers with Shared
   val component: Component = camel.context.getComponent("akka")
 
   "Endpoint url config should be correctly parsed" in {
-    val actorEndpointConfig = component.createEndpoint("akka://test/user/$a?autoAck=false&replyTimeout=987000000+nanos").asInstanceOf[ActorEndpointConfig]
+    val actorEndpointConfig = component.createEndpoint(s"akka://test/user/$$a?autoAck=false&replyTimeout=987000000+nanos").asInstanceOf[ActorEndpointConfig]
```

Comment:
This avoids a warning from scalac about "hey, it looks like you're using interpolation but forgot the s prefix!"